### PR TITLE
Enable text cursor in TableDetail

### PR DESCRIPTION
**Pull request by sasamuku**
## Summary

* `user-select: auto;` to allow selection of TableDetail text.
* `snapPoints` to be an empty array to disable snapping function

### before
https://github.com/user-attachments/assets/81edc3ca-4f2f-4773-9896-865fea6d0ce5

### after
https://github.com/user-attachments/assets/639b0a56-bde6-426b-8d5c-1b9745705f80

## Related Issue

* https://github.com/liam-hq/liam/pull/160

### CODE REVIEW

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx [hunk 1]
```diff
@@ -25,9 +25,16 @@ type Props = NodeProps<TableNodeType>
...
 export const TableNode: FC<Props> = ({ data: { table } }) => {
   const { relationships } = useDBStructureStore()
+  const [isOpen, setIsOpen] = useState(false)
+
   return (
     <>
-      <DrawerRoot direction="right">
+      <DrawerRoot
+        direction="right"
+        dismissible={false}
```
Comment (sasamuku) [inline frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx:34]: see: https://vaul.emilkowal.ski/other#non-dismissible

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx [hunk 2]
```diff
@@ -25,9 +25,16 @@ type Props = NodeProps<TableNodeType>
...
+        open={isOpen}
+        onOpenChange={setIsOpen}
```
Comment (sasamuku) [inline frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx:36]: I'm not sure, but when dismissible turn to false, set handler will be needed.
I feel this behavior is not correct 🤔 and would appreciate your review.

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx (hunk 1 above)
Comment (MH4GF) [reply frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx:34]: The close button is far away, so it looked good on dismissible, should we set it to false?

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx (hunk 2 above)
Comment (MH4GF) [reply frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx:36]: It seems to be necessary if dismissible=false!

**Review by MH4GF — COMMENTED** [review]
I've comments!!

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx (hunk 1 above)
Comment (MH4GF) [reply frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx]: nice comments 👍🏻

**Review by MH4GF — APPROVED** [review]
thanks your work!!

**Pull request merged**